e2e/plotting.spec.ts

```diff
@@ -293,7 +293,264 @@ test('plots multiple channels on the y axis and displays correct tooltips', asyn
 
   const dimensions = await chart.boundingBox();
   expect(
-    await popup.screenshot({
+    await chart.screenshot({
+      type: 'png',
+      clip: dimensions as { x; y; width; height },
+    })
+  ).toMatchSnapshot({ maxDiffPixels: 100 });
+});
+
+test('user can hide gridlines and axes labels', async ({
+  page,
+  context,
+  browserName,
+}) => {
+  // can't test this on chrome with the "have to close main window" workaround
+  if (browserName !== 'chromium') {
```

Comment:
The buttons to activate this functionality are the same as the Reset View button. The comment on line 211 seems to indicate that chrome can't properly test these features (at least until we find some way around it). Therefore, I have told chrome to skip this test altogether (or rather pass it without doing anything)

Reply:
Ah no, the issue with chrome is nothing to do with the buttons themselves! The issue is that Chrome only loads Chart.js when the main window is closed - but since React is running in the main window once we close it anything that works on the React side (e.g. the buttons) will no longer work. Since the specific Chrome test we skipped in Chrome was testing zoom (which you need Chart.js loaded for - so we need to close the main window) and then resetting the Zoom (which is done via React and so needs the main window) it can't work on Chrome. There should be nothing wrong with this test for Chrome as you can toggle the grid lines and axes labels before Chart.js actually loads!

Reply:
Gotcha, I'll have another go at it, thanks :)